app/views/govuk_publishing_components/components/feedback/_problem_form.html.erb

```diff
@@ -39,25 +39,32 @@
       } %>
 
       <%= render "govuk_publishing_components/components/button", {
-        text: t("components.feedback.send")
+        text: t("components.feedback.send"),
+        data_attributes: {
+          ga4: {
+            event_name: "form_submit",
+            type: "feedback",
+            text: t("components.feedback.send"),
+            section: t("components.feedback.help_us_improve_govuk"),
+          }
+        }
       } %>
 
-    <button
-      class="govuk-button govuk-button--secondary gem-c-feedback__close gem-c-feedback__js-show js-close-form"
-      data-track-category="Onsite Feedback"
-      data-track-action="GOV.UK Close Form"
-      aria-controls="something-is-wrong"
-      aria-expanded="true">
-      <%= t("components.feedback.close") %>
-    </button>
-
+      <button
+        class="govuk-button govuk-button--secondary gem-c-feedback__close gem-c-feedback__js-show js-close-form"
+        data-track-category="Onsite Feedback"
+        data-track-action="GOV.UK Close Form"
+        aria-controls="something-is-wrong"
+        aria-expanded="true">
```

Comment:
Is there meant to be a `form_submit` event here for the `Close` button?

Reply:
I can't see anything in the implementation guide about the Close buttons, are we meant to be tracking them as well?

Reply:
I think we may still be undecided about tracking Close buttons - it is listed in the implementation guide but there's a question around whether we do want to track them that hasn't been answered yet.

Reply:
It's listed as part of the user journeys but I can't see it listed with the specific data to send with an event, so I'd assumed we weren't tracking them.

Reply:
Cool, okey doke.

Reply:
No worries - I saw the `// 'Yes', 'No', 'Report an issue with this page', 'Send me the survey', 'Close', 'Send'` comment on the expected GTM object in the implementation guide but didn't realise Close wasn't in the table below it.